### ISSUE TITLE
si.networkInterfaces does not list all interfaces

### ISSUE THREAD

**Issue by juancampuzano** [sebhildebrandt/systeminformation#355]
**Describe the bug**
I have a computer with two network interfaces. One interface is wireless and the other is wired. What happens is that systeminformation is only bringing me the interface that I have connected to the internet.

**To Reproduce**
**Example:** When I have a single interface connected to the internet.

![Screen Shot 2020-03-30 at 6 56 29 PM](https://user-images.githubusercontent.com/52759609/78056088-22fcfa00-734a-11ea-8373-2bc23d941270.png)

**Example:** When I have the two interfaces connected
![Screen Shot 2020-03-30 at 6 56 45 PM](https://user-images.githubusercontent.com/52759609/78056107-28f2db00-734a-11ea-84b4-a5b674ce9011.png)

**Comment by sebhildebrandt** [comment]
@juancampuzano yes I am aware of this problem and need to refactor `networkInterfaces()`. Hope I will have time the next few days or weeks. Challenging time right now.

**Comment by ricardopolo** [comment]
Thank you @sebhildebrandt !!

**Comment by ricardopolo** [comment]
@sebhildebrandt any news about this one?

**Comment by juancampuzano** [comment]
> @sebhildebrandt any news about this one?

**Comment by sebhildebrandt** [comment]
@ricardopolo @juancampuzano Please have a look at the new version 6 (currently in beta). This issue should be solved in the new version. I would be happy if you can test it on your side. Please refer also to the documentation https://systeminformation.io/v6/ and https://systeminformation.io/v6/changes.html
